admin/test/system/workarea/admin/reports_system_test.rb

```diff
@@ -504,7 +504,7 @@ def test_reports_chart
           click_button t('workarea.admin.reports.timeline.add_custom_event')
         end
 
-        within '.timeline-report__event-group' do
+        within '.timeline-report' do
```

Comment:
This was failing because there were 2 on the page, but it wasn't really relevant to what we're trying to test here.